packages/fleather/lib/src/widgets/link.dart

```diff
@@ -109,6 +110,8 @@ class _CupertinoAction extends StatelessWidget {
 Future<LinkMenuAction> _showMaterialMenu(
     BuildContext context, String link) async {
   final result = await showModalBottomSheet<LinkMenuAction>(
+    clipBehavior: Clip.hardEdge,
+    useSafeArea: true,
```

Comment:
As it's been pointed out in https://github.com/fleather-editor/fleather/issues/331#issuecomment-2088701131, `useSafeArea` does not fix the issue. I tried wrapping the widget inside `builder` with `SafeArea` and seems to be working fine. Could you check it?

Reply:
As discussed, @Amir-P noticed the `useSafeArea` argument does not include padding with respect to the bottom.